live-examples/html-examples/embedded-content/iframe.html

```diff
@@ -0,0 +1,8 @@
+<iframe id="inlineFrameExample"
+    title="Inline Frame Example"
+    width="360"
```

Comment:
This is going to overflow too easily, I think 300 would be better here.

Reply:
Fixed.